README.md

```diff
@@ -76,6 +76,7 @@ The project is following Network to Code software development guidelines and are
 - pytest, coverage, and unittest for unit tests.
 
 You can ensure your contribution adheres to these checks by running `invoke tests` from the CLI.
+The command `invoke build` will create and install a docker container locally, with all dependencies to execute the tests incl. redis backend. 
```

Comment:
What about
> The command `invoke build` builds a docker container with all the necessary dependencies (including the Redis backend) locally to facilitate the execution of these tests.

Reply:
That will be easier to read. Added that to the commit. 👍🏻
This was an obstacle I encountered when I wanted to run the tests.